app/lib/conversation_details/group_details.dart

```diff
@@ -120,3 +101,32 @@ class GroupDetails extends StatelessWidget {
     );
   }
 }
+
+class _MemberTile extends StatelessWidget {
+  const _MemberTile({required this.memberId});
+
+  final UiUserId memberId;
+
+  @override
+  Widget build(BuildContext context) {
+    final profile = context.select(
+      (ContactsCubit cubit) => cubit.state.profile(userId: memberId),
+    );
+
+    return ListTile(
+      leading: UserAvatar(
+        displayName: profile.displayName,
+        image: profile.profilePicture,
+      ),
+      title: Text(
+        profile.displayName,
+        style: Theme.of(context).textTheme.labelMedium,
+        overflow: TextOverflow.ellipsis,
+      ),
+      trailing: const Icon(Icons.more_horiz),
+      onTap: () {
+        context.read<NavigationCubit>().openMemberDetails(memberId);
+      },
+    );
+  }
+}
```

Comment:
Shouldn't this (and other places that use the new cubit) be stateful and listen to changes? I'm thinking the profile might not be loaded if this is the first time it is required.